seqr/views/apis/staff_api_tests.py

```diff
@@ -0,0 +1,173 @@
+    "docs.count": "672312",
+    "store.size": "233.4mb",
+    "creation.date.string": "2019-10-03T19:53:53.846Z"
+  }]
+
+ES_CAT_ALIAS=[
+  {
+    "alias": "010203d4c1452eef3f8725bfdee23476",
+    "index": "test_index"
+  },
+  {
+    "alias": "1531a081753f4a135eef25b3496316dd",
+    "index": "1kg.vcf.gz"
+  }]
+
+TEST_INDEX_SRC_PATH="test_index_src_path"
+TEST_INDEX_SECOND_SRC_PATH="test_index_second_src_path"
+TEST_INDEX_OLD_SRC_PATH="test_index_old_src_path"
+TEST_INDEX_1KG_SRC_PATH="test_index_1kg_src_path"
+ES_INDEX_MAPPING={
+  "test_index": {
+    "mappings": {
+      "variant": {
+        "_meta": {
+          "gencodeVersion": "25",
+          "genomeVersion": "38",
+          "sampleType_1": "WES",
+          "sourceFilePath_1": "gs://seqr-datasets/v02/GRCh38/RDG_WES_Broad_Internal/v13/RDG_Broad_WES_Internal.vcf.bgz",
+          "genomeVersion_1": "38",
+          "sampleType": "WES",
+          "sourceFilePath": TEST_INDEX_SRC_PATH,
+          "gencodeVersion_1": "25"
+        },
+        "_all": {
+          "enabled": False
+        }
+      }
+    }
+  },
+  "test_index_second": {
+    "mappings": {
+      "variant": {
+        "_meta": {
+          "gencodeVersion": "25",
+          "hail_version": "0.2.24",
+          "genomeVersion": "38",
+          "sampleType_1": "WGS",
+          "sourceFilePath_1": "gs://seqr-datasets/v02/GRCh38/RDG_WGS_Broad_Internal/v10/sharded_vcf/RDG_Broad_WGS_Internal.filtered.*.vcf.gz",
+          "genomeVersion_1": "38",
+          "sampleType": "WGS",
+          "sourceFilePath": TEST_INDEX_SECOND_SRC_PATH,
+          "hail_version_1": "0.2.24",
+          "gencodeVersion_1": "25"
+        },
+        "_all": {
+          "enabled": False
+        },
+      }
+    }
+  },
+  "test_index_old": {
+    "mappings": {
+      "variant": {
+        "_meta": {
+          "gencodeVersion": "25",
+          "hail_version": "0.2.24",
+          "genomeVersion": "38",
+          "sampleType_1": "WGS",
+          "sourceFilePath_1": "gs://seqr-datasets/v02/GRCh38/RDG_WGS_Broad_Internal/v10/sharded_vcf/RDG_Broad_WGS_Internal.filtered.*.vcf.gz",
+          "genomeVersion_1": "38",
+          "sampleType": "WGS",
+          "sourceFilePath": TEST_INDEX_SRC_PATH,
+          "hail_version_1": "0.2.24",
+          "gencodeVersion_1": "25"
+        },
+        "_all": {
+          "enabled": False
+        },
+      }
+    }
+  },
+  "1kg.vcf.gz": {
+    "mappings": {
+      "variant": {
+        "_meta": {
+          "gencodeVersion": "19",
+          "genomeVersion": "37",
+          "sampleType": "WES",
+          "datasetType": "VARIANTS",
+          "sourceFilePath": TEST_INDEX_1KG_SRC_PATH
+        },
+        "_all": {
+          "enabled": False
+        },
+      }
+    }
+  }
+}
+from django.test import TestCase
+from django.urls.base import reverse
+
+from seqr.views.apis.staff_api import elasticsearch_status
+from seqr.views.utils.test_utils import _check_login
+
+class VariantSearchAPITest(TestCase):
```

Comment:
Please use a descriptive test case name - maybe `StaffAPITest`

Reply:
Thanks. Good catch. It was from a copy-paste.